deisctl/client/client.go

```diff
@@ -74,13 +74,19 @@ func (c *Client) UpgradePrep(argv []string) error {
 	usage := `Prepare platform for graceful upgrade.
 
 Usage:
-  deisctl upgrade-prep [options]
+  deisctl upgrade-prep [--stateless]
```

Comment:
Could we check to see if store services are running to know if they need to be upgraded?